shared/src/business/useCases/pendingMotion/validateDocketEntryWorksheetInteractor.ts

```diff
@@ -0,0 +1,12 @@
+import { DocketEntryWorksheet } from '@shared/business/entities/docketEntryWorksheet/DocketEntryWorksheet';
+
+export const validateDocketEntryWorksheetInteractor = ({
+  docketEntryWorksheet,
+}): Record<string, string> | null => {
```

Comment:
getFormattedValidationErrors returns null so I think we can just return from that call!